workflows/temporal/temporal.go

```diff
@@ -106,42 +113,52 @@ func (c *TemporalWF) Terminate(ctx context.Context, req *workflows.WorkflowRefer
 	return nil
 }
 
-func (c *TemporalWF) Get(ctx context.Context, req *workflows.WorkflowReference) (*workflows.StateResponse, error) {
+func (c *TemporalWF) Get(ctx context.Context, req *workflows.GetRequest) (*workflows.StateResponse, error) {
 	c.logger.Debugf("getting workflow data")
 	resp, err := c.client.DescribeWorkflowExecution(ctx, req.InstanceID, "")
 	if err != nil {
 		return nil, err
 	}
+
+	var createdAtTime time.Time
+	if resp.WorkflowExecutionInfo.StartTime != nil {
+		createdAtTime = *resp.WorkflowExecutionInfo.StartTime
+	}
+
 	// Build the output struct
 	outputStruct := workflows.StateResponse{
-		WFInfo:    workflows.WorkflowReference{InstanceID: req.InstanceID},
-		StartTime: resp.WorkflowExecutionInfo.StartTime.Format(time.RFC3339),
-		Metadata: map[string]string{
-			"task_queue": resp.WorkflowExecutionInfo.GetTaskQueue(),
-			"status":     lookupStatus(resp.WorkflowExecutionInfo.Status),
+		Workflow: &workflows.WorkflowState{
+			InstanceID:    req.InstanceID,
+			CreatedAt:     createdAtTime,
+			LastUpdatedAt: createdAtTime,
```

Comment:
For the "LastUpdatedAt" field, what're your thoughts on including more than just the time? What if we included how it was updated? For instance, we could have the time as well as the 'culprit' such as "x event was raised" or "pause/resume was called at this time"

Reply:
I think something like a "last action", or something else that gives a more description of the last thing the workflow did would be super useful. It wouldn't need to be included in `lastUpdatedAt`, though, and would probably be component-specific. My recommendation for now would be to include that kind of detailed info in the `Properties` dictionary.

Reply:
We will probably delete the temporal component code in the future given we are not using this component.